packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/tts.html

```diff
@@ -44,7 +44,7 @@ <h1 data-translate="telephony_alias_ovh_pabx_tts_title"></h1>
         class="row"
         data-ng-if="!$ctrl.loading.init && $ctrl.number.getFeatureFamily() === 'ovhPabx' && $ctrl.number.feature.isCcs()"
     >
-        <div class="col-xs-12 col-md-6 widget-presentation">
+        <div class="col-xs-12 col-md-12 widget-presentation">
```

Comment:
**suggestion:** Using a `.row > col-**-12` means that we can refer to a single `div`.